web/www/horas/Latin/Sancti/12-27.txt

```diff
@@ -225,23 +233,23 @@ $Deo gratias
 Exiit * sermo inter fratres, quod discipulus ille non moritur; et non dixit Jesus, Non moritur: sed, Sic eum volo manere, donec veniam.
 
 [Octava 2]
-!Commemoratio octavae S. Joannis.
-Ant. Iste est Joannes, qui supra pectus Domini in cena recubuit: beatus Apostolus, cui revelata sunt secreta caelestia.
+!Commemoratio Octavae S. Joannis
+Ant. Iste est Joannes, qui supra pectus Domini in caena recubuit: beatus Apostolus, cui revelata sunt secreta coelestia.
```

Comment:
The Vatican latinists agreed: the 1961 _editio typica_ has _cena_ and _cælestia_ (27 Dec., Ant. ad Benedictus, p. 291a).